clientd3d/d3drender.c

```diff
@@ -9744,6 +9761,8 @@ void SandstormInit(void)
 #define EMITTER_ENERGY	(40)
 #define EMITTER_HEIGHT	(0)
 
+	int max_position = gFullTextureSize;

```

Comment:
I don't understand the name max_position.  Is a better name possible?  Is this the max X position, or something else?